PgpWordList/PgpWordAggregationExtension.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Linq;
+
+namespace Messerli.PgpWordList
+{
+    public static class PgpWordAggregationExtension
```

Comment:
I'm not a fan of making the business code of this library into extension functions for primitive data types. The extension function on ``PgpWordAggregation` in particular seems a roundabout equivalent of putting functionality into the POD directly.

The code presented here is pretty similar to [System.BitConverter](https://docs.microsoft.com/en-us/dotnet/api/system.bitconverter?view=netframework-4.8). How about extracting this functionality into a class like `PgpWordBitConverter`?

Reply:
addressed https://github.com/messerli-informatik-ag/pgp-word-list/pull/1/commits/7ac55c2b572751086bff256d7a56e80963a28a04